core/templates/dev/head/services/explorationContextService.js

```diff
@@ -100,6 +100,21 @@ oppia.factory('explorationContextService', [
       isInExplorationEditorMode: function() {
         return (this.getPageContext() === PAGE_CONTEXT.EDITOR &&
             this.getEditorTabContext() === EDITOR_TAB_CONTEXT.EDITOR);
+      },
+
+      // Following functions help in remembering whether video is already
```

Comment:
Ok I created a new service as you suggested and included it in `/core/template/dev/head/base.html` 
Is that okay ?